neard/src/config.rs

```diff
@@ -932,18 +938,23 @@ pub fn get_genesis_url(chain_id: &String) -> String {
 pub fn download_genesis(url: &String, path: &PathBuf) {
     info!(target: "near", "Downloading config file from: {} ...", url);
 
-    let mut file = File::create(path).expect("Failed to create / write a config file.");
-    let mut easy = Easy::new();
+    let url: String = url.clone();
```

Comment:
Is this clone actually needed? Also 1) Type annotation is unnecessary 2) it is better to use `to_string()` in this case

Reply:
It is because of the move to the async function. It complains that url is not static.

Reply:
@bowenwang1996 clone is needed since we pass the value to the async scope and actix require either 'static or owned values.

Type annotation: it is here for explicitness, so we see that the value is owned.

`.to_string()` implies that you want to have a string from a non-string value, and in such place I would get confused. The explicit thing we want here is to have a copy of the value to be able to pass the ownership, so I belive `.clone()` is better.

Let's have a comment here for the future readers, that we clone the values to move ownership to actix runtime.

Reply:
Annotations removed.